### PR TITLE
Screen/onboarding

### PR DESCRIPTION
**Pull request by linyh0207**
1. Created onboarding screen (modified the BottomTabNavigator for developing)
2. Created WelcomeSwiper Component with react-native-snap-carousel 
3. Created login and signup modals

### CODE REVIEW

File: screens/OnboardingScreen.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+          <TextInput
+            label="Username"
+            value={loginUsername}
+            onChangeText={(text) => setLoginUsername(text)}
+          />
+          <TextInput
+            label="Password"
+            value={loginPassword}
+            onChangeText={(text) => setLoginPassword(text)}
+          />
```
Comment (Michael-Xie) [inline screens/OnboardingScreen.js:52]: should have the password hidden.
Check out: https://stackoverflow.com/questions/29337444/how-do-you-style-a-textinput-in-react-native-for-password-input

Comment (Michael-Xie) [reply screens/OnboardingScreen.js:52]: https://reactnative.dev/docs/textinput

File: screens/OnboardingScreen.js [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+          <TextInput
+            label="Password"
+            value={signUpPassword}
+            onChangeText={(text) => setSignUpPassword(text)}
+          />
+          <TextInput
+            label="Email"
+            value={signUpEmail}
+            onChangeText={(text) => setSignUpEmail(text)}
+          />
```
Comment (Michael-Xie) [inline screens/OnboardingScreen.js:83]: https://reactnative.dev/docs/textinput

Comment (linyh0207) [reply screens/OnboardingScreen.js:85]: Good catch! I added the secureTextEntry for password input in SignUp and Login modals.

**Pull request merged**